packages/ripple-binary-codec/package.json

```diff
@@ -38,6 +38,7 @@
   "readmeFilename": "README.md",
   "prettier": "@xrplf/prettier-config",
   "engines": {
-    "node": ">=10.22.0"
+    "node": ">=14",
```

Comment:
This is a breaking change for legacy systems. We don't need to have active support for older node versions, but we should at least allow systems to continue to use newer versions of the packages without upgrading to newer node versions.

Reply:
I am less worried about the node change as I am npm especially since 14 defaults to npm6

Reply:
@mvadari So would you like me to remove node restrictions? Should we just delete it altogether?